src/vm/prestub.cpp

```diff
@@ -1297,7 +1297,24 @@ PCODE MethodDesc::DoPrestub(MethodTable *pDispatchingMT)
 
         pCode = GetPreImplementedCode();
 
+#endif // FEATURE_PREJIT
+
+#ifdef FEATURE_READYTORUN
+        if (pCode == NULL)
+        {
+            Module * pModule = GetModule();
+            if (pModule->IsReadyToRun())
+            {
```

Comment:
I think it would be better for the profiler notifications to be in `ReadyToRunInfo::GetEntryPoint`. The debugger notifications are there as well, and there would not need to be these complex ifdefs.